src/string-bytes.c

```diff
@@ -104,6 +104,19 @@ void string_bytes_print_tpm2b(TPM2B *buffer) {
     printf("\n");
 }
 
+void string_bytes_print_tpm2b_digest(TPM2B_DIGEST *buffer) {
```

Comment:
How is this different than string_bytes_print_tpm2b()?
TPM2_DIGEST contains a TPM2B via the b field of the union, just pass that.

typedef union {				    \
	struct  {					    \
	    UINT16  size;				    \
	    BYTE    buffer[(sizeof(TPMU_HA))];			    \
	} t;						    \
	TPM2B   b;					    \
    } TPM2B_DIGEST